project-base/tests/FrontendApiBundle/Functional/Logout/LogoutTest.php

```diff
@@ -4,26 +4,16 @@
 
 namespace Tests\FrontendApiBundle\Functional\Logout;
 
-use Tests\FrontendApiBundle\Functional\Login\LoginTest;
-use Tests\FrontendApiBundle\Test\GraphQlTestCase;
+use Tests\FrontendApiBundle\Test\GraphQlWithLoginTestCase;
 
-class LogoutTest extends GraphQlTestCase
+class LogoutTest extends GraphQlWithLoginTestCase
 {
     public function testLogoutMutation()
     {
-        $isLogoutSuccess = $this->getResponseContentForQuery($this->getLogoutQuery(), [], ['HTTP_Authorization' => sprintf('Bearer %s', $this->getAccessToken())])['data']['Logout'];
+        $isLogoutSuccess = $this->getResponseContentForQuery($this->getLogoutQuery())['data']['Logout'];
```

Comment:
Is it really OK logouting user whitout authorization?

Reply:
This test now uses GraphQlWithLoginTestCase as parent class, that always logs users in :)

Reply:
Ah, ok. I missed it is in tests 🐤